components/availability/src/Availability.svelte

```diff
@@ -267,6 +268,13 @@
     }
   }
 
+  // Properties requiring further manipulation:
+  function transformPropertyValues() {
+    if (_this.start_date === null) {
```

Comment:
Should this check be only against explicit `null`? What about `undefined`?

Reply:
Good catch; will just check for a falsy value here.

(God help anybody very specifically looking for their calendar availability on January 1st 1970, passed as unix timestamp, though.)